cmd/dockerd/daemon_solaris.go

```diff
@@ -0,0 +1,69 @@
+// +build solaris
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/docker/docker/libcontainerd"
+	"github.com/docker/docker/pkg/system"
+)
+
+const defaultDaemonConfigFile = ""
+
+// currentUserIsOwner checks whether the current user is the owner of the given
+// file.
+func currentUserIsOwner(f string) bool {
```

Comment:
Theres less duplication of code this way....for now. Also, this is just a build clean PR. The other stubs (getPlatformRemoteOptions etc) will be implemented soon in upcoming functional PR's and I think it would be best to take a call on the above at that point.